src/main/java/stsjorbsmod/cards/cull/Toll.java

```diff
@@ -0,0 +1,63 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Toll extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Toll.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+    private static final int UPGRADE_PLUS_MAGIC_NUMBER = 1;
```

Comment:
It would be good to give the constants more descriptive names than "magic number"; maybe `DAMAGE_MULTIPLIER` and `UPGRADE_PLUS_DAMAGE_MULTIPLIER`?